daemon/mgr/container.go

```diff
@@ -330,6 +330,10 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		meta.Config.NetworkDisabled = true
 	}
 
+	if err := parseSecurityOpt(meta, config.HostConfig); err != nil {
```

Comment:
I think this will panic. Since in the containercreate function, HostConfig could be nil. And if you use a nil to get SecurityOpt, it would panic.

Although at the very beginning, we checked that `if config.HostConfig == nil`, we could make function `parseSecurityOpt ` more robust. How about just inputting `config.HostConfig. SecurityOpt `? Then like `func parseSecurityOpt(meta *ContainerMeta, securityOpts []string) error`